meercat/dashboard/templates/dashboard/folder_explorer.html

```diff
@@ -0,0 +1,183 @@
+
+                    <div class="" style="border:1px solid #ccc; margin-bottom:5px;">
+
+                        <textarea id="cqhelper"></textarea>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button id="patchbutton2" class="btn btn-primary" onclick="cqPatchFile();">
+                        Download Patch
+                    </button>
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div id="testModal" class="modal" tabindex="-1">
+        <div class="modal-dialog modal-xl">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Test File Helper</h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    
+                    <p>Suggestions for test file <b><i><span id="testhelpertitle">folder1/arithmetic.py</span></i></b></p>
+
+                    <div class="" style="border:1px solid #ccc; margin-bottom:5px;">
+
+                        <textarea id="testhelper"></textarea>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button id="patchbutton3" class="btn btn-primary" onclick="testPatchFile();">
+                        Download Patch
+                    </button>
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+
+
+    <br/>
+    <br/>
+    <br/>
+
+</div>
+
+
+
+<div id="myModal" class="modal" tabindex="-1">
+    <div class="modal-dialog modal-xl">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">Docstring</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+            
+                <div class="" style="border:1px solid #ccc; margin-bottom:5px;">
+                        <pre id="docstring"/>
+
+                        <!-- Add a button here that will call javascript 
+                                The javascript will use $ajax to submit yes/no to the server. 
+
+                                $.ajax({
+                                    url: '/dashboard/archeologyresults/', type: 'POST', data: { 'pr': pr, 'filename': filename }, 
+                                    success: function (result) {
+
+                                        console.log("success ajax");
+
+                                    }
+                                });
+
+                        -->
```

Comment:
Is this a TODO (in which case, maybe add a TODO comment), or dead code that can be removed?

Reply:
I removed the code.